src/transformers/models/focalnet/modeling_focalnet.py

```diff
@@ -0,0 +1,992 @@
+# coding=utf-8
+# Copyright 2022 Microsoft Research and The HuggingFace Inc. team. All rights reserved.
```

Comment:
```suggestion
# Copyright 2023 Microsoft Research and The HuggingFace Inc. team. All rights reserved.
```

Reply:
This was resolved but not addressed @alaradirik